rama-http/src/service/fs/serve_dir/mod.rs

```diff
@@ -2,6 +2,7 @@ use crate::dep::http_body::{self, Body as HttpBody};
 use crate::layer::set_status::SetStatus;
 use crate::{Body, HeaderValue, Method, Request, Response, StatusCode, header};
 use bytes::Bytes;
+use include_dir::Dir;
```

Comment:
Please add `pub use include_dir` to `dep` module in <https://github.com/plabayo/rama/blob/39039d33f9e3c93d1db406a233041b20a3557cc0/rama-http/src/lib.rs#L37> and use it also here from dep folder as that lib might be most stable so will be easier for us to upgrade that dep on major versions without breaking users (assuming they use our `dep` version).